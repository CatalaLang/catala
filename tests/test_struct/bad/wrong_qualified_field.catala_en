## Article

```catala
declaration structure Foo:
  data f content integer

declaration structure Bar:
  data f content integer
  data g content integer

declaration scope A:
  context x content Foo
  context y content integer

scope A:
  definition x equals Foo { -- f: 1 }
  definition y equals x.Foo.g
```
