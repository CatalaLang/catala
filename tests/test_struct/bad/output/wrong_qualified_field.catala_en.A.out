[ERROR] Struct Foo does not contain field g
[ERROR] 
[ERROR]   --> test_struct/bad/wrong_qualified_field.catala_en
[ERROR]    | 
[ERROR] 17 |   definition y equals x.Foo.g
[ERROR]    |                       ^^^^^^^
[ERROR]    + Article
